src/plugins/telemetry_collection_manager/server/plugin.ts

```diff
@@ -126,11 +126,10 @@ export class TelemetryCollectionManagerPlugin
     const esClient = this.getElasticsearchClient(config);
     const soClient = this.getSavedObjectsClient(config);
     // Provide the kibanaRequest so opted-in plugins can scope their custom clients only if the request is not encrypted
-    const kibanaRequest = config.unencrypted ? config.request : void 0;
```

Comment:
This is the main reason for allowing _admin_ only to retrieve the sampled data. There's no longer a need to scope the requests. It may also help us with #84183 because `kibanaRequest` won't be needed in the context anymore.